modules/core/src/v2/coins/abstractUtxoCoin.ts

```diff
@@ -1006,126 +1038,128 @@ export class AbstractUtxoCoin extends BaseCoin {
     return areAllSignaturesValid;
   }
 
-  explainTransaction(params = {} as any) {
-    const { txHex, txInfo } = params;
+  explainTransaction(params: ExplainTransactionOptions): Bluebird<TransactionExplanation> {
```

Comment:
This should also take an optional `NodeCallback<TransactionExplanation>`, then pass to `asCallback` on the coroutine. Same for all the other implementations.